server/src/main/java/org/eclipse/lsp/cobol/service/SubroutineService.java

```diff
@@ -22,6 +22,9 @@
  * filesystem load.
  */
 public interface SubroutineService {
+
+  /** ** Prefix for defining undefined subroutines */
+  String IMPLICIT_SUBROUTINE_PATH = "/implicitSubroutine/";
```

Comment:
Are we really going to support predefined subroutines?

Reply:
This is a way how we hide subroutine definitions from users in a case when users don't have any subroutine definition.
Let's say you have:
```
CALL "FOO" ....
<something>
CALL "FOO" ...
```
But don't have a FOO.cbl file. Anyway, you have several "FOO" usages, and you need to collect them. For that, we create an implicit subroutine definition and now all "FOO" usages belong to one definition.

Reply:
Well... okay.